pkl-server/src/main/kotlin/org/pkl/server/Server.kt

```diff
@@ -225,6 +225,7 @@ class Server(private val transport: MessageTransport) : AutoCloseable {
       cacheDir,
       dependencies,
       message.outputFormat,
+      false,
```

Comment:
Please do actually expose this flag in the message passing API! This will need PRs in pkl-go and pkl-swift to match.

Reply:
Roger that, I've added logic to pkl-server to pass this information along and will create complementary PRs there.